Makefile

```diff
@@ -55,19 +55,19 @@ export APP_NAME ?= Cryostat
 # Images used by the operator
 CORE_NAMESPACE ?= $(DEFAULT_NAMESPACE)
 CORE_NAME ?= cryostat
-CORE_VERSION ?= 2.3.1-snapshot
+CORE_VERSION ?= 2.3.1
 export CORE_IMG ?= $(CORE_NAMESPACE)/$(CORE_NAME):$(CORE_VERSION)
 DATASOURCE_NAMESPACE ?= $(DEFAULT_NAMESPACE)
 DATASOURCE_NAME ?= jfr-datasource
-DATASOURCE_VERSION ?= 2.3.1-snapshot
+DATASOURCE_VERSION ?= 2.3.1
 export DATASOURCE_IMG ?= $(DATASOURCE_NAMESPACE)/$(DATASOURCE_NAME):$(DATASOURCE_VERSION)
 GRAFANA_NAMESPACE ?= $(DEFAULT_NAMESPACE)
 GRAFANA_NAME ?= cryostat-grafana-dashboard
-GRAFANA_VERSION ?= cryostat-v2.3
+GRAFANA_VERSION ?= cryostat-v2.3.1
```

Comment:
I think this can really just be `2.3.1` like the others:

https://quay.io/repository/cryostat/cryostat-grafana-dashboard?tab=tags

![image](https://github.com/cryostatio/cryostat-operator/assets/3787464/a8b6571b-7923-403c-977f-df8857b270b6)